clientd3d/music.c

```diff
@@ -27,10 +27,16 @@ static Bool has_midi = False;    /* Can system play MIDI files? */
 
 static Bool playing_midi = False;  /* Is a MIDI file currently playing as an effect? */
 static Bool playing_music = False;  /* Is a MIDI file currently playing as background? */
+static Bool paused_music = False;   /* Is background music paused? */
+static Bool case_bg = False;   /* For loading midi files: which device to load to */
+static Bool badclosebool = False; /* When closing a midi and posting new 
```

Comment:
Please find a better name.

Reply:
badclosebool was removed as a variable.  Instead we are just using if (MCI Function) { Error handling } else { code predicated on the mci function }.  This part was poorly considered by me but I am slowly learning.